arcgis-ios-sdk-samples/Edit data/Update attributes (feature service)/EAOptionsViewController.swift

```diff
@@ -18,9 +18,7 @@ protocol EAOptionsVCDelegate: AnyObject {
     func optionsViewController(_ optionsViewController: EAOptionsViewController, didSelectOptionAtIndex index: Int)
 }
 
-class EAOptionsViewController: UIViewController, UITableViewDataSource, UITableViewDelegate, UIBarPositioningDelegate {
-    
-    @IBOutlet private weak var tableView: UITableView!
+class EAOptionsViewController: UITableViewController, UIBarPositioningDelegate {
```

Comment:
Conformance to `UIBarPositioningDelegate` is no longer needed. Please remove it.